content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -27,63 +27,135 @@ setup phase, thereby removing the [`NET_ADMIN` capability requirement](/docs/set
+    
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+          --set istio_cni.enabled=true | kubectl apply -f -
+    {{< /text >}}
 
-        {{< text bash >}}
-        $ helm template install/kubernetes/helm/istio-cni --name=istio-cni --namespace=istio-system | kubectl apply -f -
-        {{< /text >}}
+### Option 2: Install with Helm and Tiller via `helm install`
 
-    *  Option 2.  Install via Helm and Tiller via `helm install`.
+This option allows Helm and
+[Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components)
+to manage the lifecycle of Istio.
 
-        {{< text bash >}}
-        $ helm install install/kubernetes/helm/istio-cni --name=istio-cni --namespace=istio-system
-        {{< /text >}}
+{{< boilerplate helm-security-warning >}}
 
-    {{< tip >}}
-    Refer to the full set of `istio-cni` Helm parameters in the chart's [`values.yaml`](https://github.com/istio/cni/blob/master/deployments/kubernetes/install/helm/istio-cni/values.yaml).
-    {{< /tip >}}
+1. Make sure you have a service account with the `cluster-admin` role defined for Tiller.
+   If not already defined, create one using following command:
 
-1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/) to include the installation of the Istio CNI plugin. For example:
+    {{< text bash >}}
+    $ kubectl apply -f @install/kubernetes/helm/helm-service-account.yaml@
+    {{< /text >}}
+
+1. Install Tiller on your cluster with the service account:
 
     {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
-      --set istio_cni.enabled=true | kubectl apply -f -
+    $ helm init --service-account tiller
+    {{< /text >}}
+
+1. Install the `istio-cni` chart:
+
+    {{< text bash >}}
+    $ helm install install/kubernetes/helm/istio-cni --name istio-cni --namespace istio-system
+    {{< /text >}}
+
+1. Install the `istio-init` chart to bootstrap all the Istio's CRDs:
+
+    {{< text bash >}}
+    $ helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system
+    {{< /text >}}
+
+1. {{< boilerplate verify-crds >}}
+
+1. Select a [configuration profile](/docs/setup/kubernetes/additional-setup/config-profiles/)
+    and then install the `istio` chart corresponding to your chosen profile
+    with the `--set istio_cni.enabled=true` setting.
+    The **default** profile is recommended for production deployments.
+    Refer to the the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/#option-2-install-with-helm-and-tiller-via-helm-install)
+    to use other profiles.
+
+    {{< text bash >}}
+    $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system --set istio_cni.enabled=true
     {{< /text >}}
 
 ### Example: excluding specific Kubernetes namespaces
```

Comment:
I know this is WIP but Steve wanted "Exampled" removed

Reply:
Will do.